roles/setup_pgd/tasks/setup_pgd.yml

```diff
@@ -106,3 +106,29 @@
   ansible.builtin.import_tasks: validate_setup_pgd.yml
   when:
     - validate_only|bool or use_validation|bool
+
+# setup oidentd authentication
+- name: Start service oidentd, if not started
+  ansible.builtin.service:
+    name: oidentd
+    state: started
+
+- name: Filter items based on cs_type 'CAMO'
+  set_fact:
+    filtered_items_camo: "{{ pgd_commit_scopes | selectattr('cs_type', 'equalto', 'CAMO') | list }}"
+
+- name: Display elements of each filtered 'CAMO' item
+  ansible.builtin.include_tasks: pgd_camo_commitscope_configuration.yml
+  loop: "{{ filtered_items_camo }}"
+  loop_control:
+    label: "{{ item.cs_name }}"
+
+- name: Filter items based on cs_type 'GROUP_COMMIT'
+  set_fact:
+    filtered_items_group_commit: "{{ pgd_commit_scopes | selectattr('cs_type', 'equalto', 'GROUP_COMMIT') | list }}"
+
+- name: Display elements of each filtered 'GROUP_COMMIT' item
+  ansible.builtin.include_tasks: pgd_group_commitscope_configuration.yml
+  loop: "{{ filtered_items_group_commit }}"
+  loop_control:
+    label: "{{ item.cs_name }}"
```

Comment:
all of this needs to come prior to the `validate_setup_pgd` tasks and some tasks need to be added to `validate_setup_pgd` to validate the commit scope setup, if commit scopes were setup.